flink-streaming-scala/src/main/scala/org/apache/flink/streaming/experimental/package.scala

```diff
@@ -16,23 +16,25 @@
  * limitations under the License.
  */
 
-package org.apache.flink.contrib.streaming.scala
+package org.apache.flink.streaming.experimental
```

Comment:
To retain symmetry with the original code, would be good to have this in a `scala` subpackage.